content/ko/docs/faq/_index.md

```diff
@@ -1,498 +1,544 @@
+도커 레지스트리 프로젝트의 후속 작업입니다. 많은 주요 클라우드 벤더가
+분산 프로젝트를 제공하는 상품을 보유하고 있으며, 수많은 벤더가 동일한 제품을
+제공함에 따라 분산 프로젝트는 수년간의 개선, 보안 모범 사례, 실전 테스트의
+혜택을 받아 왔습니다.
+
+차트를 패키징하고 도커 레지스트리에 밀어내는 방법에 대한 자세한 내용은
+`helm help chart` 및 `helm help registry`를 참조하시기 바랍니다.
+
+자세한 내용은 [이 페이지](/docs/topics/registries/)를 참조하시기 바랍니다.
+
+### `helm serve` 제거
+
+`helm serve`는 개발 목적으로 머신에서 로컬 차트 저장소를
+실행했습니다. 하지만 개발 도구로서 그다지 많은 관심을 받지 못했고
+설계에도 많은 문제가 있었습니다. 결국 우리는 이것을 제거하고
+플러그인으로 분리하기로 했습니다.
+
+`helm serve`와 유사한 경험을 위해서는 [차트뮤지엄
+(ChartMuseum)](https://chartmuseum.com/docs/#using-with-local-filesystem-storage)의
+로컬 파일 시스템 스토리지 옵션과
+[servecm 플러그인](https://github.com/jdolitsky/helm-servecm)에 대해 살펴보세요.
+
+
+### 라이브러리 차트 지원
+
+헬름 3는 "라이브러리 차트"라고 하는 차트의 클래스를 지원합니다. 다른 차트에서
+공유하지만 자체 릴리스 아티팩트는 생성하지 않는 차트입니다. 라이브러리 차트의
+템플릿은 `define` 요소만 선언할 수 있습니다. 전역 범위 `define`이 아닌 콘텐츠는
+무시됩니다. 이를 통해 사용자는 여러 차트에서 재사용할 수 있는 코드 조작(snippet)을
+재사용하고 공유할 수 있으므로 중복성을 방지하고
+차트 [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)를 유지할 수 있습니다.
+
+라이브러리 차트는 Chart.yaml의 dependencies 지시어에 선언되며
+다른 차트처럼 설치 및 관리됩니다.
+
+```yaml
+dependencies:
+  - name: mylib
+    version: 1.x.x
+    repository: quay.io
+```
+
+이 기능이 차트 개발자에게 제공되는 유즈 케이스와
+라이브러리 차트를 사용하면서 생긴 모범 사례을 보게 되어
+매우 기쁩니다.
+
+### Chart.yaml apiVersion 격상
+
+라이브러리 차트 지원이 도입되고 requirements.yaml이 Chart.yaml로
+통합됨에 따라 헬름 2의 패키지 형식을 이해하고 있는 클라이언트는
+이러한 새로운 기능을 이해할 수 없게 되었습니다. 따라서 Chart.yaml의
+apiVersion을 v1에서 v2로 격상했습니다.
+
+`helm create`는 이제 이 새로운 형식을 사용하여 차트를 생성하므로
+기본 apiVersion도 격상했습니다.
+
+헬름 차트의 두 버전을 모두 지원하려는 클라이언트는 패키지 형식을 해석하는
+방법을 이해하기 위해 Chart.yaml의 `apiVersion` 필드를 검사하는 것이 좋습니다.
+
+### XDG 베이스 디렉토리 지원
+
+[XDG 베이스 디렉토리
+사양](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html)은
+파일 시스템에 설정, 데이터, 캐시된 파일을 저장할 위치를 정의하는
+이식 가능한 표준입니다.
+
+헬름 2에서 헬름은 이 모든 정보를 `~/.helm`(정확히 `helm home`이라고 함)에
+저장했으며, `$HELM_HOME` 환경 변수를 설정하여 변경하거나,
+전역 플래그 `--home` 을 사용하여 변경할 수 있습니다.
+
+헬름 3에서 헬름은 XDG 기본 디렉토리 사양에 따라 다음 환경 변수를
+고려합니다.
+
+- `$XDG_CACHE_HOME`
+- `$XDG_CONFIG_HOME`
+- `$XDG_DATA_HOME`
+
+헬름 플러그인은 `$HELM_HOME`을 스크래치패드 환경으로
+사용하려는 플러그인과의 하위 호환성을 위해
+`$XDG_DATA_HOME`의 별칭으로 `$HELM_HOME`을 여전히 전달됩니다.
+
+이러한 변화에 맞추기 위해 몇 가지 새로운 환경 변수도
+플러그인 환경에 전달됩니다.
+
+- `$HELM_PATH_CACHE` 로 캐시 경로 지정
+- `$HELM_PATH_CONFIG` 로 설정 경로 지정
+- `$HELM_PATH_DATA` 로 데이터 경로 지정
+
+헬름 3를 지원하려는 헬름 플러그인은 이 새로운 환경 변수들을
+고려해야 합니다.
+
+### CLI 명령어 이름을 변경했습니다
+
+다른 패키지 매니저의 용어와 잘 조화시키기 위해 `helm delete`를 `helm uninstall`로
+개명했습니다. `helm delete`은 여전히 `helm uninstall`의 별칭으로 유지되므로
+두 형식 중 하나를 사용할 수 있습니다.
+
+헬름 2에서는 릴리스 기록을 제거하기 위해 `--purge` 플래그를 써야
+했습니다. 이제 이 기능은 기본적으로 활성화됩니다. 예전 동작을 유지하려면
+`helm uninstall --keep-history`를 사용합시다.
+
+또한 동일한 규칙을 수용하기 위해 일부 다른 명령어들이
+개명되었습니다.
+
+- `helm inspect` -> `helm show`
+- `helm fetch` -> `helm pull`
+
+이러한 명령어는 예전 동사도 별칭으로 유지하므로
+두 가지 형식에서 계속 사용할 수 있습니다.
+
+### 네임스페이스 자동 생성
+
+존재하지 않는 네임스페이스에 릴리스를 생성할 때 헬름 2가 네임스페이스를 만들었습니다.
+헬름 3는 다른 쿠버네티스 도구의 동작을 따르며 네임스페이스가 없는 경우
+오류를 반환합니다. `--create-namespace` 플래그를 명시적으로 지정하면 헬름 3가
+네임스페이스를 만듭니다.
+
```

Comment:
We added an item that should go here: https://helm.sh/docs/faq/#what-happened-to-chartapiversion